licenses-binary/LICENSE-bouncycastle.html

```diff
@@ -0,0 +1,22 @@
+<html>
+<body bgcolor=#ffffff>
```

Comment:
Oh, can we copy the plain text version because it's simpler and Apache Spark only have plain text versions so far
- https://github.com/bcgit/bc-java/blob/r1rv80/LICENSE.md?plain=1